src/main/java/nl/nn/testtool/web/api/MetadataApi.java

```diff
@@ -64,15 +64,20 @@ public class MetadataApi extends ApiBase {
 	public Response getMetadataList(@PathParam("storage") String storageName,
 									@QueryParam("metadataNames") List<String> metadataNames,
 									@DefaultValue("-1") @QueryParam("limit") int limit,
-									@DefaultValue("") @QueryParam("filterHeader") String filterHeader,
-									@DefaultValue("(.*)") @QueryParam("filter") String filterParam ,
+									@DefaultValue("") @QueryParam("filterHeader") List<String> filterHeaders,
+									@DefaultValue("(.*)") @QueryParam("filter") List<String> filterParams ,
```

Comment:
Does the default value still work like this?

Reply:
nope, I've decided to remove it so it actually sees that the list has a length of 0. putting in an empty list as default seems to put the length at 1 for some reason